src/main/java/com/adaptavist/tm4j/jenkins/extensions/postbuildactions/TestResultPublisher.java

```diff
@@ -77,6 +83,7 @@ public void perform(Run<?, ?> run, FilePath workspace, Launcher launcher, TaskLi
             }
             logger.printf(" %s  %n", e.getMessage());
             logger.printf("%s Tests results have not been sent to Test Management for Jira %n", ERROR);
+            throw new RuntimeException();
```

Comment:
Same comment: maybe include the nested exception: `throw new RuntimeException(e);`?